mas-foundation/src/main/java/com/ca/mas/core/conf/ConfigurationManager.java

```diff
@@ -258,6 +267,12 @@ public ConfigurationProvider create(JSONObject jsonObject) throws JSONException
                 conf.setAlsoTrustPublicPki((Boolean) getValue(Config.TRUSTED_PUBLIC_PKI, jsonObject, Boolean.FALSE));
                 continue;
             }
+
+            if (attr == Config.SSL_PINNING_ENABLED) {
+                conf.setAlsoTrustPublicPki((Boolean) getValue(Config.TRUSTED_PUBLIC_PKI, jsonObject, Boolean.FALSE));
```

Comment:
dont we have to check the value from the config value ?

Reply:
when value is unavailable it will be overridden by the default value.